src/general_physics.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+from src.common_constants import CommonConstants
+from src.vec2d import Vec2d
+
+
+@dataclass(frozen=True)
+class GeneralPhysics:
+    pixel_to_meter_ratio: float = CommonConstants.PIXEL_TO_METER
+    meter_to_pixel_ratio: float = CommonConstants.METER_TO_PIXEL
+    timestep_size: float = 1 / 60
+
+    def pixel_to_meter(*args) -> float:
+        if type(args[0]) == float and len(args) == 1:
+            res = args[0] * GeneralPhysics.pixel_to_meter_ratio
+        elif type(*args) == Vec2d:
+            res = GeneralPhysics.pixel_to_meter_ratio * args[0]
```

Comment:
Why changing the order of the factors? Makes it more difficult to read. Or has this a reason I don't get?

Reply:
Aligned it.